src/components/views/auth/ServerConfig.js

```diff
@@ -211,16 +212,37 @@ export default class ServerConfig extends React.PureComponent {
         return setTimeout(fn.bind(this), this.props.delayTimeMs);
     }
 
-    showHelpPopup = () => {
-        const CustomServerDialog = sdk.getComponent('auth.CustomServerDialog');
-        Modal.createTrackedDialog('Custom Server Dialog', '', CustomServerDialog);
+    showHsHelpPopup = () => {
+        Modal.createTrackedDialog('What does this mean?', 'HS', InfoDialog, {
+            title: _t("Homeserver URL"),
+            description: _t(
+                "You can use the custom server options to sign into other " +
+                "Matrix servers by specifying a different homeserver URL. This " +
+                "allows you to use %(brand)s with an existing Matrix account on a " +
+                "different homeserver.",
+                { brand: SdkConfig.get().brand },
+            ),
+            button: _t("Dismiss"),
+        });
+    };
+
+    showIsHelpPopup = () => {
+        Modal.createTrackedDialog('What does this mean?', 'IS', InfoDialog, {
+            title: _t("Identity Server URL"),
+            description: _t(
+                "The optional identity server is responsible for allowing you " +
+                "to find people by their e-mail address.",
```

Comment:
and phone number

Reply:
https://github.com/vector-im/element-web/issues/3711

Reply:
well, it also handles your pending 3pid invites - it's not just discovery.